src/lib/routes/admin-api/client-metrics.ts

```diff
@@ -74,8 +78,12 @@ class ClientMetricsController extends Controller {
                 openApiService.validPath({
                     operationId: 'getFeatureUsageSummary',
                     tags: ['Metrics'],
+                    summary: `Last hour of usage and a list of applications that have reported seeing this feature toggle`,
+                    description:
+                        'Separate counts for yes (enabled), no (disabled), as well as how many times each variant was selected during the last hour',
```

Comment:
Can we make the summary shorter? Much of that information can (and should) be moved into the description.

Is this an accurate representation?

```suggestion
                    summary: 'Get recent metrics for a feature'
                    description: 'Returns evaluation data and a list of applications that have reported seeing this feature toggle within the last hour. The response contains separate counts for yes (enabled), no (disabled), as well as how many times each variant was selected.'
```